improver_tests/acceptance/test_feels_like_temp.py

```diff
@@ -53,6 +53,7 @@ def test_basic(tmp_path):
                    for param in params]
     output_path = tmp_path / "output.nc"
     args = [*input_paths,
-            "--output", output_path]
+            "--output", output_path,
+            "--model-id-attr", "mosg__model_configuration"]
```

Comment:
I believe --output has to be the last argument, but I could be wrong.

Reply:
Might be stylistically, but it doesn't seem to break anything.  Having named arguments means the order is independent.  Happy to change it if you want consistent style though.

Reply:
No it's fine, I'm not sure why I got the idea the order mattered for clize, but I had some issues.